components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.scss

```diff
@@ -0,0 +1,159 @@
+@import "~styles/variables";
+@import "@carbon/grid/scss/grid";
+
+:host {
+  @extend .bx--grid;
+  position: absolute;
+  top: calc(100% + 8px);
+  right: 0;
+  padding: 16px;
+  border-radius: 8px; // not variable
+  background: $chef-white;
+  box-shadow: 8px 8px 24px 0 rgba(197, 199, 209, 0.6);
+  width: 100%;
+  transform-origin: top right;
+
+  @include carbon--breakpoint-up(md) {
+    max-width: 467px;
+  }
+
+
+  * {
+    font-family: 'Inter Regular';
```

Comment:
not in the system yet so I should probably pull this but also our current font doesn't work with the expected stacking of the buttons because the text is too long.

Reply:
Drive-by, haven't looked closely at this PR yet but components we build should be flexible enough to handle different font types (when possible of course, don't have full context).